core/esmf-aspect-meta-model-java/src/test/java/org/eclipse/esmf/aspectmodel/resolver/AspectModelResolverTest.java

```diff
@@ -49,6 +49,9 @@ public void testLoadDataModelExpectSuccess() throws URISyntaxException {
          final AspectModel result = new AspectModelLoader( urnStrategy ).load( testUrn );
          final Resource aspect = createResource( TestModel.TEST_NAMESPACE + "Test" );
          assertThat( result.mergedModel().listStatements( aspect, RDF.type, SammNs.SAMM.Aspect() ).nextOptional() ).isNotEmpty();
+         assertThat( result.files().get( 0 ).headerComment() ).isNotNull();
+         assertThat( result.files().get( 0 ).headerComment().get( 0 ) ).contains(
+               "# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH" );
```

Comment:
I think the header comment lines should not contain `# `. Please adjust the tests (and implementation as necessary).